content/en/platform/corda/4.6/enterprise/release-notes-enterprise.md

```diff
@@ -18,25 +18,15 @@ weight: 1
 
 ## Corda Enterprise 4.6.6
 
-Corda Enterprise 4.6.6 is a patch release of Corda Enterprise that fixes an urgent security issue - CVE-2021-44228 - caused by the Apache Log4j 2 dependency.
-
 {{< warning >}}
-
-Upgrade now to avoid exposure to the [Apache Log4j 2 vulnerability to attack](https://nvd.nist.gov/vuln/detail/CVE-2021-44228). This is the most secure way to mitigate any risks associated with this vulnerability.
-
+Patch 4.6.6 contains dependency Log4j 2.15.0. A new vulnerability has been discovered in version 2.15.0 of the log4j logging library, as described here: https://nvd.nist.gov/vuln/detail/CVE-2021-45046. Apache have now released version 2.16.0 of the library to address the issue. Corda Enterprise 4.8.5 is due for release December 17 2021.
 {{< /warning >}}
 
-### Upgrade recommendation
-
-As a developer, you should upgrade to the [latest released version of Corda](../../../../../en/platform/corda/4.8/enterprise.html) as soon as possible. The latest Corda Enterprise release notes are on this page, and you can find the latest upgrade guide [here](../../../../../en/platform/corda/4.8/enterprise/upgrading-index.md).
+Corda Enterprise 4.8.4 is a patch release of Corda Enterprise that attempted to fix an urgent security issue - CVE-2021-44228 - caused by the Apache Log4j 2 dependency.
```

Comment:
Is the random '2' meant to be there?

Reply:
And 'attempted to fix'. Did it not?

Reply:
Yes, meant to be there, no it did not fix it. 4.8.5 will...